src/utils/windowUtils.ts

```diff
@@ -41,6 +41,30 @@ export module Utils {
       }
     }
 
+    /**
+     * To be used in the first line of a deprecated method
+     *
+     * @param {string} version The version when the tagged method became obsolete
+     */
+    export function deprecated(version: string) {
```

Comment:
It may be useful to have an optional `message` parameter so we can say "Please use xMethod instead"